examples/from-file-weighted/profile.json

```diff
@@ -22,8 +22,8 @@
       "constraints": [
         {
```

Comment:
Again not really the code you are touching but it might be a good idea to mention the ``` --set-from-file-directory``` flag in the related example readmes. 

Something like 

>An example of loading a set of values from a file.
In this case, the names are populated from the file `greek.csv`.
To run this example, the jar must be placed in this directory.
Alternativly the  --set-from-file-directory command line arguement can be used with the path to this directory eg. 
```--set-from-file-directory=C:/path/to/dataHelix/examples/from-file/```